docs/releaseNotes.yml

```diff
@@ -56,6 +56,41 @@ items:
   - version: 2.3.0
     date: '2022-06-06'
     notes:
+      - title: Envoy xDS v2 API removal, removal of ECMAScript regexes and v2 transport protocols
```

Comment:
Why are these in this PR? They should have already been merged?

Reply:
No? I wanted to get them in to 2.3, but because of `regex_type: unsafe` we couldn't.

Reply:
But the changelog shouldn't mention v2 transport protocol removal, as this PR doesn't do that.

Reply:
Oh, but this is in the wrong version's section.